wrappers/services_test.go

```diff
@@ -470,3 +470,71 @@ func (s *servicesTestSuite) TestStartSnapMultiServicesFailStartCleanup(c *C) {
 		{"show", "--property=ActiveState", svc1Name},
 	})
 }
+
+func (s *servicesTestSuite) TestServiceAfterBefore(c *C) {
```

Comment:
The test is very nice (and smart) - I would also like to have an additional "dumb" test that just uses a very simple before and after and validates against a fully written [servce] snippet. I often look^Wsometimes^Wevery-now-and-then at tests when I want to know details about e.g. generated files and that seems to be useful. Also it gives an easy overview about what goes in and what comes out without having to parse a regexp :)